pallets/validators-manager/src/mock.rs

```diff
@@ -364,36 +356,12 @@ parameter_types! {
 
 impl CandidateTransactionSubmitter<AccountId> for TestRuntime {
     fn submit_candidate_transaction_to_tier1(
```

Comment:
What happened here? Is this function no longer needed then? Is it a placeholder now?